controllers/Sources.php

```diff
@@ -25,7 +25,7 @@ public function show() {
 
         // load sources
         $sourcesDao = new \daos\Sources();
-        echo '<button class="source-add">' . \F3::get('lang_source_add') . '</button>' .
+        echo '<button class="source-add">' . \F3::get('lang_source_add') . '</button>' .' or '.
```

Comment:
Having or in the buttons is not very nice but adding it here is even worse, since it cannot be translated.